### PR TITLE
feat : added list component

### PR DESCRIPTION
**Pull request by abhishek17551**
Screenshot (124)
Screenshot (125)

### CODE REVIEW

**Review by saifudinbadani — APPROVED** [review]
Stacked list looks really good.

File: pages/components/lists/list.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../../../styles/styles.css">
+    <link rel="icon" href="../../../assets" type="image/icon type" />
```
Comment (ark2002) [inline pages/components/lists/list.html:9]: adviced to add a proper title name instead of keeping it document

**Review by ark2002 — APPROVED** [review]
looks goood to me

File: pages/components/lists/list.html (hunk 1 above)
Comment (abhishek17551) [reply pages/components/lists/list.html:9]: will modify title for all the pages at once. thanks.

**Pull request merged**